### PR TITLE
Windows

### PR DESCRIPTION
**Pull request by mattn**
But currently, doesn't support echo-back.

### CODE REVIEW

File: sshd_posix.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import (
+	"io"
+	"os/exec"
+	"syscall"
+	"unsafe"
+
+	"github.com/kr/pty"
+)
+
+type ShellFile File
```
Comment (hnakamur) [inline sshd_posix.go]: @mattn I think this line should be
```
type ShellFile os.File
```

**Review by hnakamur — CHANGES_REQUESTED** [review]
@mattn Thanks for your pull request!
I think we need a fix to avoid the build error:
https://travis-ci.org/hnakamur/go-sshd/jobs/206052108#L172

**Comment by mattn** [comment]
Ooops. Sorry.

**Comment by hnakamur** [comment]
@mattn Thanks for your fix. I'm afraid the build still fails. Could you fix it?
https://travis-ci.org/hnakamur/go-sshd/jobs/206191100

**Comment by mattn** [comment]
雑でしたすみません ><

**Comment by mattn** [comment]
もすこしお待ちを

**Comment by hnakamur** [comment]
@mattn ビルド通ったのでマージします。ありがとうございました！

**Comment by mattn** [comment]
ありがとうございます

**Pull request merged**